flocker/docs/bootstrap/_extension.py

```diff
@@ -0,0 +1,60 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Bootstrap extension entry point.
+"""
+
+from docutils import nodes
+from sphinx.writers.html import HTMLTranslator
+from . import _simple, _tabs
+
+
+def setup(app):
+    """
+    Entry point for sphinx extension.
+    """
+
+    for module in [_simple, _tabs]:
+        module.setup(app)
+
+
+class HTMLWriter(HTMLTranslator):
+    """
+    Overrides part of the default HTMLTranslator to provide specific
+    class names on some generated HTML elements.
```

Comment:
I think one method was copied from Sphinx's `HTMLTranslator` and the other one from the docutils base class that Sphinx's class inherits. They were then modified to add a couple of class names to the table tag and to wrap cell content in paragraph tags. As per my other comment below, if you want to change/improve this, please do. Otherwise we can add a comment attributing the original source.